assignment.js

```diff
@@ -32,6 +32,12 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+let baseNumber = 2
+
+for (i = 0; i <= 12; i++) {
+    console.log(baseNumber * i)
```

Comment:
@justfinish be careful with indentation here. This line is over-indented. While this won't impact how the program runs - it can result in code being harder to read.

Reply:
@justfinish nevermind, it looks like you are using 4 spaces to represent 1 level of indentation. While that's ok - that isn't what's widely practice in the industry. Most companies use 2 space per 1 level of indentation for JS.